.github/runtime.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+      "name": "rococo",
+      "package": "rococo-parachain-runtime",
+      "path": "runtime/rococo",
+      "uri": "wss://rpc.rococo-parachain.litentry.io:443"
+    },
+    {
+      "name": "litmus",
+      "package": "litmus-parachain-runtime",
+      "path": "runtime/litmus",
+      "uri": "wss://rpc.litmus-parachain.litentry.io:443"
+    },
+    {
+        "name": "litentry",
```

Comment:
"uri" is the key to determinate whether one entry is valid or not?
Besides, small indentation issue here.

Reply:
yes

thanks - I'll fix it